src/services/member/plugins/data/service.ts

```diff
@@ -0,0 +1,188 @@
+import { MailerDecoration } from '../../../../plugins/mailer';
+import { UnauthorizedMember } from '../../../../utils/errors';
+import { Repositories } from '../../../../utils/repositories';
+import FileService from '../../../file/service';
+import { Item } from '../../../item/entities/Item';
+import { Actor } from '../../entities/member';
+import {
+  actionArraySchema,
+  appActionArraySchema,
+  appDataArraySchema,
+  appSettingArraySchema,
+  itemArraySchema,
+  itemCategoryArraySchema,
+  itemFavoriteArraySchema,
+  itemLikeArraySchema,
+  itemMembershipArraySchema,
+  messageArraySchema,
+  messageMentionArraySchema,
+} from './schemas/schemas';
+import {
+  anonymizeMentionsMessage,
+  anonymizeMessages,
+  getFilteredData,
+} from './utils/anonymize.utils';
+import { RequestDataExportService } from './utils/export.utils';
+
+export class DataMemberService {
```

Comment:
I just noticed the path of the files. Could we change it to `plugins/export` or `plugins/export-data`? `plugins/data` is way to generic, I thought it was fixtures for tests.
Perhaps also change the service name to `ExportMemberDataService`